src/ops.rs

```diff
@@ -25,6 +25,7 @@ use std;
 use std::fs;
 use std::net::{Shutdown, SocketAddr};
 #[cfg(any(unix))]
+use std::ops::Deref;
 use std::os::unix::fs::PermissionsExt;
```

Comment:
Move `#[cfg(any(unix))]` above this line, to avoid windows trying to import unix only extensions

Reply:
My bad, sorry!